addons/web_editor/views/snippets.xml

```diff
@@ -448,18 +448,18 @@
         <div class="o_we_image_shape">
             <we-row string="Shape" class="o_we_full_row">
                 <we-select-pager data-name="shape_img_opt" class="o_we_select_grid o_we_fake_transparent_background">
-                    <we-select-page string="Basics" t-if="not no_animations">
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_1" data-select-label="Square 1" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_2" data-select-label="Square 2" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_3" data-select-label="Square 3" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_4" data-select-label="Square 4" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_5" data-select-label="Square 5" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_6" data-select-label="Square 6" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_rounded_1" data-select-label="Square Rounded 1" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_square_rounded_2" data-select-label="Square Rounded 2" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_1" data-select-label="Organic Soft" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_2" data-select-label="Organic Medium" data-animated="true"/>
-                        <we-button data-set-img-shape="web_editor/basic/bsc_organic_3" data-select-label="Organic Hard" data-animated="true"/>
+                    <we-select-page class="o_we_basic_shapes" string="Basics">
```

Comment:
Why did you move the `t-if` to the individual buttons? IIRC it was on the parent because otherwise we could end up with an empty page, since we have to hide the _whole page_ if we don't allow animations.

Reply:
Indeed, we don't want an empty page and that's why it was on the whole page. But I added 3 shapes in Basics that we *do* want to show in mass_mailing. (if we hide the whole page, we can't see the three basic shapes I added). 
see https://github.com/odoo/odoo/pull/95974/files/81eb4cf0eae7bf9d8796a90f29a9a8001b45dc5b#diff-b26221e9411fb6f3725ce5a30c4659a22d81a9a69f666bdbd6daa77e03ec3253R350

Reply:
I see. Does it work to remove the t-if on `we-select-page` with your xpath? So that we don't end up with an empty page in the contexts where your xpath is not active?

I mean you'd leave the t-ifs you added on each button, put back the t-if on we-select-page, and then in your xpath remove that t-if from we-select-page. I don't know in which order things are done and so I'm not sure you can remove a t-if that way but I think it's worth a shot. @dmo-odoo What do you think?